cmd/sdkch/main.go

```diff
@@ -252,8 +283,8 @@ func writeEntryFile(filename string, bs []byte) {
 		log.Fatal(err)
 	}
 
-	log.Printf("Unreleased changelog entry written to: %s\n", filename)
```

Comment:
I'm using `log.Printf()` here cause this informative output goes to stderr instead of stdout

Reply:
k I'll reverse